src/node-explorer-provider.ts

```diff
@@ -56,12 +58,81 @@ export class NodeExplorerProvider
     this.registerRefresh();
     this.registerOpenDocsLink();
     this.registerDownloadCommand();
+    this.pollForUpdates();
   }
 
   getTreeItem(element: PeerBaseTreeItem): vscode.TreeItem {
     return element;
   }
 
+  private async getPeers() {
+    this.previousStatus = this.currrentStatus;
+    this.currrentStatus = await this.ts.getPeers();
+    return this.currrentStatus;
+  }
+
+  private async diffRelay() {
```

Comment:
I'm not sure if this is the best way to implement this function -- feedback appreciated!

Reply:
I think this is a sensible approach. 👍